### ISSUE TITLE
Add NAICS Code Recommendations to NAICS Task

### ISSUE THREAD

**Issue by aaronvux** [newjersey/navigator.business.nj.gov#2443]
As Poppy, I need the ability to associate a NAICS Code to my business so that I have accurate records.

## Acceptance Criteria
- Update Industry to NAICS codes mapping
- Serve all available NAICS codes recommendations based on mapping of Industry (except Generic) from Onboarding in a Radio button list
- If no recommendations for the user's industry exist, display the links and input box.
- Selecting a recommended NAICS code displays the Select This NAICS Code button
  - Clicking the button displays success message with contextual link
  - Display NAICS Code with edit link.
  - Clicking the Edit link displays the previous recommended list
- Clicking Search for other NAICS Codes displays content with links to Siccode.com and Naics.com with Input box and Save NAICS code button.

## Additional Acceptance Criteria
- Automatically change status to In-Progress WHEN a radio button is clicked
- Automatically change status to Completed WHEN a NAICs code has been saved
- 508 Compliant
- Follows Design
- Works in Mobile

## Design
- Desktop
**Default**
https://www.figma.com/file/IKcrTIyvaS593LuJbAVFR3/Navigator-Designs-(Sprint-33%2B)?node-id=12311%3A248478

**Selection**

**Confirmation**

**Search**

**Select Code**

**Success**

- Mobile

## Analytics
TBD

## Approvals
- [ ] Content Review
- [ ] UX Review
- [ ] PO Approval

**Comment by aaronvux** [comment]
Comments from Jess:
1. Add NAICS code to the Profile.
2. Analytics - track NAICS codes through Analytics (user profiles?)

Create new story for only allowing users to search and input their NAICS code.  Recommendations will be separate work.

**Comment by Jesslax** [comment]
@aaronvux bumping the need for the user story above. Needs to happen before this task.

**Comment by aaronvux** [comment]
Revisit to ensure proper sizing at planning.

**Comment by aaronvux** [comment]
@mikewolfd @aloverso the spreadsheet Veronica's team has created will have the NAICS description.  The description is in the design, but we don't call it out specifically in the AC.  Should I add it?

**Comment by aaronvux** [comment]
@mikewolfd may need to slightly adjust the AC to show the first 10 recommendations, then a show more link to display the rest. Let's discuss at planning.  @carye3 @fiorella-io @Jesslax

**Comment by aaronvux** [comment]
LGTM @fiorella-io @Jesslax please review.

**Comment by fiorella-io** [comment]
@mikewolfd Can we please adjust the following on mobile 320px:
- [ ] Make sure content doesn't extend outside its container
- [ ] radio buttons need to align with content
- [ ] content should wrap
![image](https://user-images.githubusercontent.com/91498500/168121470-35439bb5-61af-45d0-aaea-e338fe44bde8.png) - <img width="262" alt="image" src="https://user-images.githubusercontent.com/91498500/168121530-042346b0-b750-4691-8089-d34a90ce9591.png">

For desktop:
- [ ] indent the radio button options
![image](https://user-images.githubusercontent.com/91498500/168122646-d0858045-671f-4fb1-9c49-d43963269dc1.png) - <img width="340" alt="image" src="https://user-images.githubusercontent.com/91498500/168122708-c40aaf3d-50cb-48ee-a863-8617fbdafd9b.png">
